### ISSUE TITLE
Getting 422 error

### ISSUE THREAD

**Issue by unional** [changesets/action#190]
https://github.com/justland/just-web/runs/6894968941?check_suite_focus=true

I think I have it setup correctly. I have other repos working. Not sure what is the problem, maybe some server issue?

**Comment by Andarist** [comment]
Ye, it's hard to tell - it is likely a problem with the GitHub API or something. I don't think it's actionable on our side - unless we can prove that the problem is on our side. I will keep an eye for similar reports to see if this is a common issue or not.

**Comment by unional** [comment]
```
creating pull request
o [HttpError]: Validation Failed: {"resource":"PullRequest","code":"custom","message":"No commits between main and changeset-release/main"}
    at /home/runner/work/_actions/changesets/action/v1/dist/index.js:718:926
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async Object.k [as runVersion] (/home/runner/work/_actions/changesets/action/v1/dist/index.js:918:5763)
    at async /home/runner/work/_actions/changesets/action/v1/dist/index.js:920:2283 {
  status: 422,
  headers: {
    'access-control-allow-origin': '*',
Error: Validation Failed: {"resource":"PullRequest","code":"custom","message":"No commits between main and changeset-release/main"}
    'access-control-expose-headers': 'ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Used, X-RateLimit-Resource, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, X-GitHub-SSO, X-GitHub-Request-Id, Deprecation, Sunset',
    connection: 'close',
    'content-length': '237',
    'content-security-policy': "default-src 'none'",
    'content-type': 'application/json; charset=utf-8',
    date: 'Wed, 15 Jun 2022 07:26:34 GMT',
    'referrer-policy': 'origin-when-cross-origin, strict-origin-when-cross-origin',
    server: 'GitHub.com',
    'strict-transport-security': 'max-age=31536000; includeSubdomains; preload',
    vary: 'Accept-Encoding, Accept, X-Requested-With',
    'x-content-type-options': 'nosniff',
    'x-frame-options': 'deny',
    'x-github-media-type': 'github.v3; format=json',
    'x-github-request-id': '07C1:5FAB:2FAF3A:959DAD:62A989AA',
    'x-ratelimit-limit': '1000',
    'x-ratelimit-remaining': '944',
    'x-ratelimit-reset': '16[55](https://github.com/justland/just-web/runs/6894968941?check_suite_focus=true#step:7:56)279129',
    'x-ratelimit-resource': 'core',
    'x-ratelimit-used': '[56](https://github.com/justland/just-web/runs/6894968941?check_suite_focus=true#step:7:57)',
    'x-xss-protection': '0'
  },
  request: {
    method: 'POST',
    url: 'https://api.github.com/repos/justland/just-web/pulls',
    headers: {
      accept: 'application/vnd.github.v3+json',
      'user-agent': 'octokit-core.js/3.1.0 <environment undetectable>',
      authorization: 'token [REDACTED]',
      'content-type': 'application/json; charset=utf-8'
    },
    body: `{"base":"main","head":"changeset-release/main","title":"Version Packages","body":"This PR was opened by the [Changesets release](https://github.com/changesets/action) GitHub action. When you're ready to do a release, you can merge this and the packages will be published to npm automatically. If you're not ready to do a release yet, that's fine, whenever you add more changesets to main, this PR will be updated.\\n\\n\\n# Releases"}`,
    request: { agent: [Agent], hook: [Function: bound bound r] }
  },
  errors: [
    {
      resource: 'PullRequest',
      code: 'custom',
      message: 'No commits between main and changeset-release/main'
    }
  ],
  documentation_url: 'https://docs.github.com/rest/reference/pulls#create-a-pull-request'
}
```

For the record, this is the error. (as the GitHub Action log will be deleted after a while). 🌷 

Also a side note: the publishing step is working. I do `changeset version` locally and push the changes to get around this error.

**Comment by krzkaczor** [comment]
I am running into the same problem. This broke with pnpm 7. I think it has to be related somehow but I am still investigating.

**Comment by unional** [comment]
```
> changeset publish

🦋  info npm info clibuilder
🦋  info clibuilder is being published because our local version (8.0.3) has not been published on npm
🦋  info Publishing "clibuilder" at "8.0.3"
🦋  error 
🦋  error packages failed to publish:
🦋  clibuilder@8.0.3
 ELIFECYCLE  Command failed with exit code 1.
Error: The process '/home/runner/setup-pnpm/node_modules/.bin/pnpm' failed with exit code 1
Error: The process '/home/runner/setup-pnpm/node_modules/.bin/pnpm' failed with exit code 1
    at m._setResult (/home/runner/work/_actions/changesets/action/v1/dist/index.js:102:7258)
    at m.CheckComplete (/home/runner/work/_actions/changesets/action/v1/dist/index.js:102:6686)
    at ChildProcess.<anonymous> (/home/runner/work/_actions/changesets/action/v1/dist/index.js:102:5723)
    at ChildProcess.emit (events.js:314:20)
    at maybeClose (internal/child_process.js:1022:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:287:5)
    ```
    
    Another failure. Hard to see what is the problem.

**Comment by unional** [comment]
Funning into it again.

**Comment by DiamondThree** [comment]
I had the same problem as you and don't know if you solved it. https://github.com/DiamondThree/test-changeset/actions/runs/3798304202/jobs/6459913884

**Comment by DiamondThree** [comment]
I fixed it, I guess that .yml  problem, I deleted the pull_request config. for example:
<img width="398" alt="image" src="https://user-images.githubusercontent.com/24630517/209929494-969367fe-418e-47dc-b584-5155c912d68b.png">

**Comment by unional** [comment]
For me that's not the case. I am using share config and only [clibuilder](https://github.com/unional/clibuilder) is having problem right now.

e.g. [type-plus](https://github.com/unional/type-plus) is working fine

**Comment by bluwy** [comment]
I think it's a bit hard to follow-up on what happened here that it happened a long while ago. I'll go ahead and close this but feel free to open a new one if it happens again.
